incus/profile: Add support for creating profile from yaml

awalvie opened the pull request:
Ref: #741

stgraber commented:
I updated the commit title a bit and aligned the commit author with the Signed-off-by line so the test should clear now. I also added the `make i18n` result to a `i18n: Update translation templates` commit which should then clear the rest of the failures.

Assuming tests behave now, I'll merge this and you can repeat much of the same with the remaining commands, thanks!

awalvie commented:
> I updated the commit title a bit and aligned the commit author with the Signed-off-by line so the test should clear now. I also added the make i18n result to a i18n: Update translation templates commit which should then clear the rest of the failures.

Thank you! I'll take care of them in the follow-up PRs.
